projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.ts

```diff
@@ -54,32 +55,43 @@ export class ItemDetailViewComponent implements DetailRecord, OnInit, OnDestroy
 
   /**
    * Constructor
-   * @param recordService - RecordService
-   * @param loanService - LoanService
+   * @param _recordService - RecordService
+   * @param _loanService - loanService
+   * @param _itemService - ItemsService
    */
   constructor(
-    private recordService: RecordService,
-    private loanService: LoanService
+    private _recordService: RecordService,
+    private _loanService: LoanService,
+    private _itemService: ItemsService
   ) {}
 
   ngOnInit() {
-    this._recordObs = this.record$.subscribe( record => {
-      this.record = record;
-      const numberOfRequest$ = this.loanService.numberOfRequests$(record.metadata.pid);
-      const locationRecord$ = this.recordService.getRecord('locations', record.metadata.location.pid);
-      forkJoin([numberOfRequest$, locationRecord$]).subscribe(
-        ([numberOfRequest, location]) => {
-          this.numberOfRequests = numberOfRequest;
-          this.location = location;
-          this.recordService.getRecord('libraries', extractIdOnRef(location.metadata.library.$ref)).subscribe(
-            library => this.library = library
-          );
-        }
-      );
+    this._recordObs = this.record$.subscribe(record => {
+      this._loadInfo(record);
+    });
+    this._itemService.updateItem$.subscribe((itemPid: string) => {
```

Comment:
This should be destroyed in the ngOnDestroy method. You can use an array of subscribers.